1_OCaml/TP/TP4.ipynb

```diff
@@ -22,12 +22,83 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.                else decomposition n q in (decomposition n parties);;
</pre><p>Passer le n (decomposition n parties);; à la ligne pour que ce soit plus clair, mais bien pour le reste.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>